librosa/core/audio.py

```diff
@@ -471,7 +473,7 @@ def to_mono(y):
 
 @cache(level=20)
 def resample(
-    y, orig_sr, target_sr, res_type="kaiser_best", fix=True, scale=False, **kwargs
+    y, orig_sr, target_sr, *, res_type="kaiser_best", fix=True, scale=False, **kwargs
```

Comment:
Not sure if this one should kwarg the sampling rates.  Normally I would because they're both values that could be of the same type and therefore easily confused.  I don't particularly like these parameter names though.  Maybe something like:

```python
def resample(y, *, sr, new_sr, ...)
```
would be more clear?